recipes/scevan/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: scevan
+  version: 1.0.0  # Replace with the desired version
+
+source:
+  url: https://github.com/AntonioDeFalco/SCEVAN/archive/refs/tags/1.0.0.tar.gz  # Replace with the GitHub URL of the release
+  sha256: cc14b6b80019e06181bc69ab6f54bde9878b3b525645f925626bfed28df655fd  # Generate the checksum using 'sha256sum' command
+
+build:
+  number: 0
+  noarch: generic  # Specify that the package is noarch
+  run_exports: '{{ pin_compatible("SCEVAN", max_pin="x.x") }}'
+  rpaths:
+    - lib/R/lib
+    - lib
+  script: "$R -e \"install.packages(c('Rcpp', 'remotes', 'devtools'), lib='$PREFIX/lib/R/library', repos='https://cran.rstudio.com/')\""
+
+requirements:
+  host:
+    - r
+    - r-base
+    - r-remotes
+    - r-devtools
+  run:
+    - r
+    - r-base
+    - r-devtools
+    - r-paralleldist
+    - r-pheatmap
+    - r-forcats
+    - r-dplyr
+    - bioconductor-fgsea
+# - r-yaGST - manually with devtools
```

Comment:
Why this dependency is left to be (pre-)installed by the user ?
I think it should be a proper runtime dependency and lines https://github.com/bioconda/bioconda-recipes/pull/45649/files#diff-180caf3dc4ce15be03b03267239715f1249fa8f1d7e031a38dc7624e5189ef07R46-R48 / https://github.com/bioconda/bioconda-recipes/pull/45649/files#diff-180caf3dc4ce15be03b03267239715f1249fa8f1d7e031a38dc7624e5189ef07R27 could be removed

Reply:
You may need a build script like https://github.com/bioconda/bioconda-recipes/pull/45645/files#diff-0d601fcf99e217658acf5443fb7fb9fe3d9691742cee85bccdb60bcfbdb200acR2 to install the library locally and test it without `devtools`

Reply:
could not a way to install "yaGST" other than with `devtools::install_github()`. "yaGST" is not on CRAN or Bioconductor or conda

Reply:
I see!
Maybe a recipe should be added for `yaGST` first ...
I'll let the Bioconda maintainers to decide what would be the best!

Reply:
I would suggest that this gets shipped as it is and eventually when someone adds that additional recipe, then it could be added as conda dependency here? Otherwise it is harsh on contributors to have to chase every single dependent package. There are other packages that do installations of deps via devtools.